packages/amazon-pay-utils/src/amazon-pay-v2-script-loader.ts

```diff
@@ -0,0 +1,37 @@
+import { ScriptLoader } from '@bigcommerce/script-loader';
+
+import {
+    PaymentMethod,
+    PaymentMethodClientUnavailableError,
+} from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+import {
+    AmazonPayV2HostWindow,
+    AmazonPayV2InitializeOptions,
+    amazonPayV2Regions,
+    AmazonPayV2SDK,
+} from './amazon-pay-v2';
+
+export default class AmazonPayV2ScriptLoader {
+    constructor(
+        private scriptLoader: ScriptLoader,
+        private windowWithAmazonPay: AmazonPayV2HostWindow = window,
+    ) {}
+
+    async load(method: PaymentMethod<AmazonPayV2InitializeOptions>): Promise<AmazonPayV2SDK> {
+        const { initializationData } = method;
+        const region = initializationData?.region || 'us';
```

Comment:
The rest looks good to me.
Just this one point, when could `region` value miss during initialization? 
Should we consider this case as an error instead of failling silently?

Reply:
hm, good question.
As I see in BCApp region should be always. But need more clarifications with BE developers.
And current default value for region was used before by some reasons - [https://github.com/bigcommerce/checkout-sdk-js/pull/2396/files#diff-c6696f657e6d98be7a08527c3cf41f16aff4d549587c00a5136b192abbc46455L15](https://github.com/bigcommerce/checkout-sdk-js/pull/2396/files#diff-c6696f657e6d98be7a08527c3cf41f16aff4d549587c00a5136b192abbc46455L15)

So to not broke something while migration process. I propose to make refactoring and add changes for default values in separate PR. And maybe move this default value to BE side if it is really needed.